client/admin_test.go

```diff
@@ -129,6 +129,13 @@ func Example_permission() {
 	context := testutils.NewRootTestBaseContext()
 	client := client.NewAdminClient(context, s.ServingAddr(), client.Permission)
 
+	// The test server creates a permission config entry for 'server.TestUser'.
+	// Delete it first so it does not interfere with our configs.
+	err := client.Delete(server.TestUser)
```

Comment:
awkward to have this in example code. How about

```go
s := server.StartTestServerFor("myuser", nil)
```

instead, and then the example can take the user into account?